### ISSUE TITLE
Failed to build androidpayload on windows

### ISSUE THREAD

**Issue by degipanju** [rapid7/metasploit-payloads#525]
I try to build on windows using command: **mvn clean package -Dandroid.sdk.path=/path/to/android-sdk -Dandroid.release=true -P deploy**, but then I get this following error:

> [ERROR] Failed to execute goal com.simpligility.maven.plugins:android-maven-plugin:4.4.3:proguard (default-proguard) on project Metasploit-AndroidPayload: : MojoExecutionException: ANDROID-040-001: Could not execute: Command = cmd.exe /X /C ""C:\Program Files\Java\jdk-11.0.11\bin\java" -Xmx512M -jar C:\Users\XXX\\.m2\repository\net\sf\proguard\proguard-base\5.2.1\proguard-base-5.2.1.jar @E:\metasploit\metasploit-payloads\java\androidpayload\app\target\proguard\temp_config.cfg", Result = 1 -> [Help 1]

Any suggestion?

**Comment by timwr** [comment]
Did you replace /path/to/android-sdk with the actual SDK location?
I haven't actually tried building on windows, you're definitely better off using Linux/Macos/docker to build it.

**Comment by green-force** [comment]
> Did you replace /path/to/android-sdk with the actual SDK location? I haven't actually tried building on windows, you're definitely better off using Linux/Macos/docker to build it.

Yes, the actual location is C:\Users\XXX\AppData\Local\Android\Sdk which is the default sdk location of Android Studio. I will try to build on linux. Can I build it using latest java and maven version?

**Comment by green-force** [comment]
I have tried building on linux but still got similar error

![Screenshot](https://user-images.githubusercontent.com/60676243/150275919-f2816cd8-bf30-46d0-96ef-579cfbd6303f.png)

I used latest jdk and maven version

![Screenshot from 2022-01-20 11-58-52](https://user-images.githubusercontent.com/60676243/150277832-e45fb7c2-1c91-4ff4-8013-f8087dfc5067.png)

Need some help
